pkg/volume/fc/fc_util.go

```diff
@@ -386,11 +387,12 @@ func (util *FCUtil) DetachBlockFCDisk(c fcDiskUnmapper, mapPath, devicePath stri
 		glog.Errorf("fc: last error occurred during detach disk:\n%v", lastErr)
 		return lastErr
 	}
-
-	// The volume was successfully detached from node. We can safely remove the loopback.
-	err = volumeutil.BlockVolumePathHandler.RemoveLoopDevice(blkUtil, loop)
-	if err != nil {
-		return fmt.Errorf("fc: failed to remove loopback :%v, err: %v", loop, err)
+	if loop != "" {
```

Comment:
len(loop) == 0 (and for other changes too)

Reply:
done